src/components/skiplink-list/skiplink-list.scss

```diff
@@ -40,6 +40,12 @@
   }
 }
 
+.sbb-link-list-title {
```

Comment:
title is never visible. does this make any sense at all? Maybe to not read an empty label? Can you test behavior?

Reply:
Semantically speaking there is a difference between not visible and not displayed and I feel like that should be preserved. But anyway, I'll test the label behaviour and see what happens

Reply:
As you suggested, the title is ignored if empty.
We can both remove the CSS rule and the conditional `set ID` on the next feedback